src/VSMSBuildExtensions/VSMSBuildExtensions.proj

```diff
@@ -61,7 +61,7 @@
         <DestinationPath>$(OutputPath)/%(VSMSBuildExtensionsContent.DeploymentSubpath)%(RecursiveDir)%(Filename)%(Extension)</DestinationPath>
       </VSMSBuildExtensionsContent>
       <VSMSBuildExtensionsContent Include="$(ArtifactsBinDir)Microsoft.DotNet.MSBuildSdkResolver\$(Configuration)\net472\SdkResolvers\Microsoft.DotNet.MSBuildSdkResolver\Microsoft.DotNet.MSBuildSdkResolver.xml"
-        DestinationPath="$(OutputPath)/SdkResolvers/Microsoft.DotNet.MSBuildSdkResolver/Microsoft.DotNet.MSBuildSdkResolver.xml" />
+        DestinationPath="$(OutputPath)/MSBuildSdkResolver/Microsoft.DotNet.MSBuildSdkResolver.xml" />
```

Comment:
It was difficult to get this one file that kind of bucks the patterns in the rest of the file to work, so I ejected entirely and hand-coded the DestinationPath (for build output usage) and later the `PackagePath` (for nuget packaging purposes).

Reply:
I'd promote this to a comment in the file for posterity.